submodules/eks/README.md

```diff
@@ -13,8 +13,8 @@
 
 | Name | Version |
 |------|---------|
-| <a name="provider_aws"></a> [aws](#provider\_aws) | >= 4.0 |
-| <a name="provider_null"></a> [null](#provider\_null) | >= 3.1.0 |
+| <a name="provider_aws"></a> [aws](#provider\_aws) | 4.32.0 |
```

Comment:
How did this get pinned ?

Reply:
it gets generated based on whatever it evals to at generation which is odd but how it works.

Reply:
it should look at the provider constraint which is `>= 4.0` and not pin to a single version